manifests/resource/mailhost.pp

```diff
@@ -1,5 +1,7 @@
 # define: nginx::resource::mailhost
 #
+# lint:ignore:140chars
+#
```

Comment:
Can we just wrap these lines @ 140 char rather than ignoring the lint warnings? I realize it's just docs, but...

If you indent the following line one more space
```
# blah blah
#   blah
```
it will get continued as one lines in the YARD doc, I think. Also, we'll switch to Puppet strings for param docs at some point anyway.